server/src/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -688,10 +689,6 @@ public UserVm resetVMPassword(ResetVMPasswordCmd cmd, String password) throws Re
 
         if (result) {
             userVm.setPassword(password);
-            // update the password in vm_details table too
-            // Check if an SSH key pair was selected for the instance and if so
-            // use it to encrypt & save the vm password
-            encryptAndStorePassword(userVm, password);
```

Comment:
So, the password will not be encrypted anymore in the DB? Is that it?

Reply:
encryptAndStorePassword was already done inside resetVMPasswordInternal.